pipelines/upgrade/04-install_proxy.yaml

```diff
@@ -14,7 +14,7 @@
       - "--foreman-proxy-foreman-base-url https://{{ server_fqdn.stdout }}"
       - "--foreman-proxy-oauth-consumer-key {{ oauth_consumer_key.stdout }}"
       - "--foreman-proxy-oauth-consumer-secret {{ oauth_consumer_secret.stdout }}"
-      - "--foreman-proxy-content-parent-fqdn {{ server_fqdn.stdout }}"
+      - "{{ '' if (pipeline_version == 'nightly' or 2.3 is version(pipeline_version, '>')) else ('--foreman-proxy-content-parent-fqdn %s' % server_fqdn.stdout) }}"
```

Comment:
Pipeline version is the *target* version of the upgrade pipeline. You probably want `forklift_upgrade_version_start` here

Reply:
Ok will update... though by the time we have reached this line actually needing this check, we should be dropping it all together.

Reply:
The previous version would have omitted the param on 3.17 (to 18 to nightly) upgrade runs, and that would be wrong.